contracts/cw-named-groups/src/contract.rs

```diff
@@ -222,19 +193,34 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
 }
 
 fn query_dump(deps: Deps) -> StdResult<DumpResponse> {
-    // Create dump of all groups.
-    let dump = GROUPS
+    let mut groups_map: HashMap<String, Vec<String>> = HashMap::new();
```

Comment:
No need for hash map here too. It will use hashing internally for keys. Just use `Vec<(String, Vec<String)>`